src/gui/widgets/Fader.cpp

```diff
@@ -198,16 +198,15 @@ void Fader::mousePressEvent( QMouseEvent* mouseEvent )
 
 void Fader::mouseDoubleClickEvent( QMouseEvent* mouseEvent )
 {
-	bool ok;
-	float newValue;
+	bool ok = false;
 	// TODO: dbV handling
-	newValue = QInputDialog::getDouble( this, tr( "Set value" ),
-				tr( "Please enter a new value between %1 and %2:" ).
-						arg( model()->minValue() * m_conversionFactor ).
-						arg( model()->maxValue() * m_conversionFactor ),
-					model()->getRoundedValue() * m_conversionFactor,
-					model()->minValue() * m_conversionFactor,
-					model()->maxValue() * m_conversionFactor, model()->getDigitCount(), &ok ) / m_conversionFactor;
+	float newValue = QInputDialog::getDouble(this, tr("Set value"),
+						 tr("Please enter a new value between %1 and %2:")
+							 .arg(model()->minValue() * m_conversionFactor)
+							 .arg(model()->maxValue() * m_conversionFactor),
+						 model()->getRoundedValue() * m_conversionFactor, model()->minValue() * m_conversionFactor,
+						 model()->maxValue() * m_conversionFactor, model()->getDigitCount(), &ok)
+		/ m_conversionFactor;
```

Comment:
Looks better to me this way, what do you think?
```suggestion
	auto minv = model()->minValue() * m_conversionFactor;
	auto maxv = model()->maxValue() * m_conversionFactor;
	float newValue = QInputDialog::getDouble(this, tr("Set value"),
			tr("Please enter a new value between %1 and %2:").arg(minv).arg(maxv),
			model()->getRoundedValue() * m_conversionFactor,
			minv, maxv, model()->getDigitCount(), &ok)
		/ m_conversionFactor;
```